### DISCUSSION TITLE
Errors while trying to install Deepin Graphics Drivers Manager

### DISCUSSION THREAD

**Discussion by gabaozin** [linuxdeepin/developer-center#7521] [Q&A | 问答板块]
Hi, I have found this app Deepin Graphics Drivers Manager and wanted to install it.

Instructions at the github file are incomplete for a layman.

Searching on the web I was able to find this article with more detailed instructions:

https://riyadulmollick.medium.com/install-deepin-graphics-driver-manager-on-deepin-20-5052dfe876bc

Unfortunately, when trying the command make, after doing cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr .. # Install to /usr (which I don't know if I should change this DCMAKE_INSTALL_PREFIX or not. I have tried which dcmake, but could not find it)

I get the following error:

/opt/apps/deepin-graphics-driver-manager/deepin-graphics-driver-manager/server/main.cpp:47:5: error: ‘logger’ was not declared in this scope; did you mean ‘dlogger’?
   47 |     logger->registerAppender(consoleAppender);
      |     ^~~~~~
      |     dlogger
make[2]: *** [server/CMakeFiles/deepin-graphics-driver-manager-service.dir/build.make:132: server/CMakeFiles/deepin-graphics-driver-manager-service.dir/main.cpp.o] Erro 1
make[1]: *** [CMakeFiles/Makefile2:202: server/CMakeFiles/deepin-graphics-driver-manager-service.dir/all] Erro 2
make: *** [Makefile:136: all] Erro 2

Do you think you can help?

I'm running Deepin 23 on a lenovo Legion laptop of CPU 12th Gen Intel(R) Core(TM) i7-12700H (please let me know if I should inform something else about my pc configuration).

Thank you,

Gabriel

**Comment by BLumia** [comment]
Deepin Graphics Drivers Manager is a project for v15 era of DDE, which I believe no longer works well with v20 and later version of DDE.

The error you got is because DTK changed some of its API so it needs to adapt to the newer version of DTK to build. But even though you managed to fix all the build errors, it's probably still not going to work as intended as it did in v15.

HTH
